main.go

```diff
@@ -2,12 +2,15 @@ package main
 
 import (
 	"fmt"
+	"net/http"
 	"os"
 
 	"github.com/andreykaipov/goobs"
 	"github.com/spf13/cobra"
 )
 
+var version = "0.2.1-dev"
```

Comment:
Not a fan of setting this manually, as it's disconnected from the release process and tagging. We can set this via `go build` and keep a default value of "unknown". I'll push a few more changes to this branch on your fork, if you don't mind.

Reply:
I changed the code to have the version set to an empty string. In that case the request header will just be set to "obs-cli" without any version. During build you can set the version number if you like:
```
go build -ldflags="-X main.version=0.2.0"
```